model/model.py

```diff
@@ -35,34 +36,32 @@ def get_current_floor(self) -> int:
         pass
 
 
-
 class Bellhop(BellhopModelInterface):
 
-    def __init__(self, num_floors: int, capacity: int):
+    def __init__(self, level: Level):
+        self.level = level
         self._curr_state: State = State.WAIT_INPUT
```

Comment:
Hadn't noticed the type-hinting here before, kinda like it, can we add similar to the Level class?
Also we should make a style choice here on what to do with the line above, we've type-hinted on the parameter but not on setting the field, my main thing would be to be consistent, happy to leave as is.

Reply:
@thbenschop w.r.t hinting for setting variables, how serious were you about "type hints everywhere"

Reply:
I agree, leave off this/make a reasonable best-effort until someone invests more time to really understand how mypy works

Reply:
btw for reference mypy does check for it getting set to a different type, so have added hints to all class variables (that don't have pygame types)